terraform/azure/modules/hashistack/hashistack.tf

```diff
@@ -100,7 +100,7 @@ resource "azurerm_public_ip" "hashistack-server-public-ip" {
   name                         = "hashistack-server-ip-${count.index}"
   location                     = "${var.location}"
   resource_group_name          = "${azurerm_resource_group.hashistack.name}"
-  public_ip_address_allocation = "static"
+  allocation_method            = "static"
```

Comment:
```suggestion
  allocation_method            = "Static"
```

Reply:
Thanks for the quick point out. Let me commit the changes